uiowa/uiowa
# Display config changes for person field updates

- Resolves #3847
- Updates in preparation for SPPA launch
- Fixes Grad College thesis defense view permissions regression 

# How to test

Check that a default site syncs with no config weirdness. View a person page--no changes or errors should be present.

Sync amcs.uiowa.edu, and check that it syncs properly.
Check any people from https://amcs.local.drupal.uiowa.edu/graduates, and check that any new fields from #3789 are visible (dissertation topic/thesis title, defense date, current position institution and title, and advisor(s))

## Stories 
- As an SPPA student, I can filter the opportunities page (https://sppa.local.drupal.uiowa.edu/opportunities), click an opportunity, hit the back button and my filters are still preserved.
- As a content editor on the grad college site I can see and edit thesis defense content on the content admin page (https://grad.local.drupal.uiowa.edu/admin/content?title=&type=thesis_defense).
